server/app.ts

```diff
@@ -0,0 +1,23 @@
+import express from 'express';
+import { join } from 'path';
+import compression from 'compression';
+import cors from 'cors';
+import route from './routes';
+
+require('env2')('.env');
+
+const app:express.Application = express();
+app.set('port', process.env.PORT || 5001);
+app.use(cors());
+app.use(express.json());
+app.use(express.urlencoded({ extended: false }));
+app.use(compression());
+app.use('/api', route);
+const { NODE_ENV } = process.env;
+if (NODE_ENV === 'production') {
```

Comment:
I think there is no need for this if condition, because we want a server to run same in the `Development` environment and `Production` environment